oeq-ts-rest-api/src/Search.ts

```diff
@@ -0,0 +1,203 @@
+import * as Common from './Common';
+import { is } from 'typescript-is';
+import {GET} from './AxiosInstance';
```

Comment:
destructuring is inconsistent

```suggestion
import { is } from 'typescript-is';
import { GET } from './AxiosInstance';
```

Reply:
Yeah, really need to get this setup with linting and formatting. Just with the plan to move to own repo I want to wait until then rather than messing things up here. So, I guess it's manual for now - which I do by manually triggering Prettier in my IDE as I work.